spec/lib/generators/form_generator_spec.rb

```diff
@@ -0,0 +1,94 @@
+require 'spec_helper'
+
+require 'generators/rom/form_generator'
+
+describe ROM::Generators::FormGenerator do
+  destination File.expand_path('../../../../tmp', __FILE__)
+
+  before(:each) do
+    prepare_destination
+  end
+
+  specify "a create form" do
+    run_generator ['users', '--command=create']
+
+    default_adapter = ROM.adapters.keys.first
+
+    expect(destination_root).to have_structure {
+      directory 'app' do
+        directory 'forms' do
+          file 'new_user_form.rb' do
+            contains <<-CONTENT.strip_heredoc
+              class NewUserForm < ROM::Model::Form
+                commands users: :create
+
+                input do
+                  set_model_name 'User'
+
+                  # define form input attributes
+                  # attribute :name, String
+
+                  timestamps :created_at, :updated_at
```

Comment:
`timestamps` helper w/o args defaults to those, so we can simply generate a call to this helper w/o args :)